src/previews.ts

```diff
@@ -18,6 +19,7 @@ export const previews: PreviewFlags = {
   rtdbmanagement: false,
   functionsv2: false,
   golang: false,
+  purgegcr: false,
```

Comment:
This will all be subject to API council changes, but IMO, the open-sesame flag should just match whatever the name of the command is in this case.

Reply:
Change to deletegcfartifacts per API review